rpcclient/roundrobin.go

```diff
@@ -0,0 +1,367 @@
+// Copyright (c) 2015-2017 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package rpcclient
+
+import (
+	"errors"
+	"sync"
+
+	"github.com/btcsuite/websocket"
+)
+
+var (
+	// ErrNoConnAvailable indicates no Conn is available for pick().
+	ErrNoConnAvailable = errors.New("no Connection is available")
+	// ErrNoUsableConnAvailable indicates no usable connections available i.e.
```

Comment:
Need a blank line before for code consistency.

Reply:
Fixed it.